src/sentry/static/sentry/app/views/projectPluginDetails.jsx

```diff
@@ -79,9 +81,19 @@ class ProjectPluginDetails extends AsyncView {
   renderActions() {
     const {pluginDetails} = this.state;
     const enabled = this.getEnabled();
+    const {pluginId} = this.props.params;
 
+    let disabled = false;
+    if (DEPRECATED_PLUGINS.includes(pluginId)) {
+      disabled = true;
+    }
```

Comment:
@MeredithAnya I know I am making a fuss over something small but couldn't we just make this line:
```
const disabled = DEPRECATED_PLUGINS.includes(pluginId);
```
Consider it an optional suggestion